src/mame/drivers/bwidow.cpp

```diff
@@ -674,6 +676,14 @@ static INPUT_PORTS_START( spacduel )
 	/* bit 7 is tied to a 3kHz clock */
 	PORT_BIT( 0x80, IP_ACTIVE_HIGH, IPT_CUSTOM ) PORT_READ_LINE_MEMBER(bwidow_state, clock_r)
 
+	/* Cabinet type.  The cocktail's harness has a jumper between pins
+	   R and P, which maps to bit 7 of address 0x0907 */
+
```

Comment:
You can use CONFNAME, CONFSETTING for things that are not DIP switches.

Reply:
Fixed!